x-pack/plugin/old-lucene-versions/src/main/java/org/elasticsearch/xpack/lucene/bwc/codecs/lucene70/Lucene70Codec.java

```diff
@@ -0,0 +1,15 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+package org.elasticsearch.xpack.lucene.bwc.codecs.lucene70;
+
+public class Lucene70Codec extends BWCLucene70Codec {
+
+    public Lucene70Codec() {
+        super("Lucene70");
```

Comment:
This is a thin wrapper around the BWC Codec so that the `Lucene70` name continues to operate. Notice, we also have a `BWCLucene70Codec`, which arises from when Lucene 9.x shipped with its own `Lucene70Codec`/`Lucene70` ( ES carried a different name)